rainier-scalacheck/src/main/scala/com/stripe/rainier/scalacheck/package.scala

```diff
@@ -50,4 +50,11 @@ object `package` {
     arbitrary[A].flatMap(a =>
       arbitrary[Real].flatMap(density => RandomVariable(a, density)))
 
+  implicit def cogenGenerator[A](implicit CA: Cogen[A],
```

Comment:
The Comonad tests needed these. I'm not familiar enough with the stuff above to see if this new code is helpful with existing stuff.